packages/bbui/src/Form/Core/Checkbox.svelte

```diff
@@ -1,22 +1,23 @@
-<script>
+<script lang="ts">
   import "@spectrum-css/checkbox/dist/index-vars.css"
   import "@spectrum-css/fieldgroup/dist/index-vars.css"
   import { createEventDispatcher } from "svelte"
+  import { ChangeEventHandler } from "svelte/elements"
 
   export let value = false
-  export let id = null
-  export let text = null
+  export let id: string | undefined = undefined
+  export let text: string | undefined = undefined
   export let disabled = false
   export let readonly = false
-  export let size
+  export let size: string = "M"
```

Comment:
You can probably take this further and type this as `"S" | "M" | "L"`. Not sure if XS and XL are valid options here but they might be as well - would need to check the CSS file imported at the top here.